lib/tsconfig/3.8.x/ng9/README.md

```diff
@@ -0,0 +1,20 @@
+<h1 align="center">
+   code style - TypeScript 3.8.x + Angular 9
+</h1>
+
+## About
+
+This sub-package hosts the [TypeScript](https://www.typescriptlang.org/) configuration for TypeScript 3.5.x. and Angular 9.
```

Comment:
I think it is important to mention somewhere in this doc that [TypeScript 3.8.x is only supported as from Angular 9.1.0](https://github.com/angular/angular/blob/master/CHANGELOG.md#910-2020-03-25)

Reply:
You're right, I will add a comment about this. Thanks for your remark

Reply:
I changed a bit the main README file and the README files under "3.8.x/ng9" and "3.5.x/ng8".
What do you think ? :blush:

Reply:
Alright, much better :wink: